pom.xml

```diff
@@ -355,6 +355,11 @@
       <artifactId>caffeine</artifactId>
       <version>3.1.1</version>
     </dependency>
+    <dependency>
+      <groupId>com.opentable.components</groupId>
+      <artifactId>otj-pg-embedded</artifactId>
+      <version>1.0.1</version>
+    </dependency>
```

Comment:
Know what you're saying, if we want to use jancekpetr we _could_ upgrade the reference to the `otj-pg-embedded` there in that lib to 1.0.1. But the lib hasn't been updated for 5 years. Seems like it would be a lot better at this point to seriously consider converting over to testcontainers.